src/Components/ValidationCaisse.tsx

```diff
@@ -0,0 +1,367 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from "react";
+import LandingPage from "./LandingPage";
+import { useNavigate, useParams } from "react-router-dom";
+import { Toolbar } from "primereact/toolbar";
+import { Dropdown, DropdownChangeEvent } from "primereact/dropdown";
+import { DataTable } from "primereact/datatable";
+import { Column } from "primereact/column";
+import { Panel } from "primereact/panel";
+import { Fieldset } from "primereact/fieldset";
+import { Button } from "primereact/button";
+import { MultiSelect, MultiSelectChangeEvent } from "primereact/multiselect";
+import { InputText } from "primereact/inputtext";
+import { SplitButton } from "primereact/splitbutton";
+
+interface Product {
+  jour?: string;
+  piece?: string;
+  reference?: string;
+  facture?: string;
+  type?: number;
+  compte_general?: string;
+  compte_tiers?: string;
+  libelle_ecriture?: string;
+  date_echeance?: string;
+  position_journal?: string;
+  debit?: string;
+  credit?: string;
+}
+interface VisibilityProps {
+  setProduct?: Dispatch<SetStateAction<Product>>;
+  product?: Product;
+  products?: Product[];
+  setProducts?: Dispatch<SetStateAction<Product[]>>;
+  setSelectedProduct?: Dispatch<SetStateAction<Product>>;
+  selectedProduct?: Product;
+}
+
+interface Type {
+  name?: string;
+  code: number | null;
+}
+const typeope: Type[] = [
+  { name: "Tous", code: null },
+  { name: "Sortie", code: 1 },
+  { name: "Entrées", code: 2 },
+  { name: "Fond de Caisse", code: 3 },
+  { name: "Transfert de caisse", code: 4 },
+  { name: "Versement bancaire", code: 5 },
+];
+const caisse: Type[] = [{ name: "Tous", code: null }];
+const sortie: Type[] = [{ name: "Tous", code: null }];
+const TypeCaisse: Type[] = [{ name: "Tous", code: null }];
+
+const mvt: Type[] = [
+  { name: "10100000 - Capital", code: null },
+  { name: "10400000 - Primes liées au capital social", code: 1 },
+  { name: "10500000 - Ecart de réévaluation", code: 2 },
+];
+
+const personne: Type[] = [
+  { name: "Atabe", code: 1 },
+  { name: "SIAKAM", code: 2 },
+];
+
+const notif: Type[] = [
+  { name: "Aucun", code: null },
+  { name: "Mail", code: 2 },
+  { name: "Sms", code: 3 },
+];
+const produit: Product[] = [
```

Comment:
why you don't make that in file call like database/fake_data.tsx in order to easily reuse that.

Reply:
it's planned